### DISCUSSION TITLE
Ability to limit file logging

### DISCUSSION THREAD

**Discussion by kurbans** [TychoTheTaco/Twitch-Drops-Bot#90] [Ideas]
Would it be possible to include a configuration to limit file logging? I've just started running this on my Synology NAS and realized that at the rate it's going that logging file is going to grow pretty quickly. Additionally, the extra noise when it's writing to the drives is becoming pretty bothersome. 

Thanks for the awesome work on this project!

**Comment by TychoTheTaco** [comment]
Yes, I will add an option for it.
